docker-build-proto/Dockerfile

```diff
@@ -0,0 +1,46 @@
+FROM golang:1.14-alpine3.11 as builder
+
+RUN apk add --update --no-cache build-base curl git upx && \
+  rm -rf /var/cache/apk/*
+
+ENV GOLANG_PROTOBUF_VERSION=1.3.1 \
+  GOGO_PROTOBUF_VERSION=1.3.1
+RUN GO111MODULE=on go get \
```

Comment:
`GO111MODULE` can be removed, I believe it is enabled by default in Go 1.14.

Reply:
without it this error: `go: cannot use path@version syntax in GOPATH mode` pops up